elasticapm/contrib/django/client.py

```diff
@@ -149,7 +150,12 @@ def get_data_from_request(self, request, event_type):
                         self.logger.debug("Can't capture request body: %s", str(e))
                         data = "<unavailable>"
                 if data is not None:
-                    result["body"] = data
+                    # This can be removed once we figure out a soluction to
+                    # https://github.com/elastic/apm-agent-python/issues/305
+                    if len(str(data)) > constants.KEYWORD_MAX_LENGTH:
```

Comment:
There is https://github.com/elastic/apm/blob/main/specs/agents/field-limits.md#long_field_max_length-configuration in the specs. Is that something that is better done as part of #305?

Reply:
Ah, I had not realized that it was extended to more than `db.statement` -- I found the old `apm-dev` thread but didn't realize it had been made applicable to request body. I'll update and add that config.